components/fxa-client/src/internal/config.rs

```diff
@@ -90,46 +90,6 @@ impl Config {
         self
     }
 
-    // FIXME
-    #[allow(clippy::too_many_arguments)]
-    pub(crate) fn init(
```

Comment:
Took me a while to understand what was happening here - was `StateV1` the only consumer of this method, and hence it can now be removed as dead code?

Reply:
Yup! Clippy was complaining that it was dead code, so it went